group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -67,14 +68,22 @@ public enum ConsumerGroupState {
 
         private final String name;
 
+        private final String capitalizedName;
```

Comment:
I would rather do it the other way around in order to be consistent with the classic group states. Could we capitalize the states in L63-67 and lower case them here?

Reply:
Done.